internal/redfishwrapper/task.go

```diff
@@ -76,13 +74,13 @@ func (c *Client) ConvertTaskState(state string) constants.TaskState {
 	switch strings.ToLower(state) {
 	case "starting", "downloading", "downloaded", "scheduling":
 		return constants.Initializing
-	case "running", "stopping", "cancelling":
+	case "running", "stopping", "canceling":
```

Comment:
I don't think these should change as they are States that correspond to the gofish library. https://github.com/stmcginnis/gofish/blob/4b51c15f0a3df649821f9fc395a82483ff60bb2a/redfish/task.go#L16-L55 

It is spelled `Cancelling` in that library. We should probably be using the const from that package anyway.

Reply:
uggh good catch will back out. We are using the consts from that package already. The problem is we are converting that package's types <-> string when it should be done by that package itself (via stringer imo).

Reply:
should be back to original (plus `//nolint:` directive)